app/javascript/utilities/waitForDataLoaded.js

```diff
@@ -0,0 +1,10 @@
+export function waitForDataLoaded() {
```

Comment:
For some reason I had a hard time naming this, and still not loving it too much. Suggestions for renaming this are welcome if anyone has any alternatives they like better than this 😅

Reply:
Refactor (suggestion): Maybe `waitOnUserData`?

Side-note: This would be great to have this reused where we currently use setInterval explicitly. For another PR though.

Reply:
I like the idea! But given that we're actually waiting for the `fetchBaseData` ([from here](https://github.com/forem/forem/blob/main/app/assets/javascripts/initializers/initializeBodyData.js#L12-L34)) I renamed it to `waitOnBaseData`, I think it fits better that the original `waitForDataLoaded` for sure 😄